src/minigames/dragMinigame.ts

```diff
@@ -0,0 +1,59 @@
+import { Minigame } from "./minigame";
+import { Application, Assets, FederatedMouseEvent, Graphics, Sprite, Texture } from "pixi.js";
```

Comment:
nit: I think some of these are not being used, e.g. Application, Graphics. Can we remove them?